Misc/NEWS.d/next/Library/2023-11-08-23-32-03.gh-issue-111835.ufFiuW.rst

```diff
@@ -0,0 +1,3 @@
+The :class:`mmap.mmap` class now has an :meth:`~mmap.mmap.seekable` method
+that can be used where it requires a file-like object with seekable. Patch
```

Comment:
You should also document the seek() method change here (just copy the What's New text).